.eslintrc.js

```diff
@@ -22,24 +22,19 @@ const defaultRules = {
   'no-caller': 2,
   'no-console': 2,
   'no-else-return': 0,
-  'no-empty-character-class': 2,
   'no-multi-spaces': 2,
   'no-param-reassign': 0,
```

Comment:
Valid point! I simply didn't think about it in this PR and you're prolly right that they date back long ago. :-)

They're ripe for reviewing+removing. 👍

Reply:
Aight! Removed all these in https://github.com/Trustroots/trustroots/pull/1126/commits/a5b5135eb256f8cb6e95a21fe1ddefd81162aa54 and flipped  `no-strict` from `0` to `2` since it was relevant for this PR; https://github.com/Trustroots/trustroots/pull/1126/commits/8456afdfdb3a2eb5e7a639fcfa505c1145a22708